CONTRIBUTING.md

```diff
@@ -16,5 +16,6 @@ Giving some love to PHP CS
 ---------------------------
 
 ```bash
-bin/php-cs-fixer fix ./
+composer require --dev 'friendsofphp/php-cs-fixer:^2.0'
```

Comment:
@mcg-web how about creating a composer script to install and run the php cs?
```
composer cs-fix
```

Could even do a `composer test` as well for the phpunit call

Reply:
I'm not against, we will dedicate a PR to this